app/assets/javascripts/utilities/modal-workflow.js

```diff
@@ -0,0 +1,37 @@
+/* global $ */
```

Comment:
Something I learnt last week is that for many situations there are eslint-env settings for different global variables. 

So this could be defined `/* eslint-env jQuery */` as per: https://eslint.org/docs/user-guide/configuring#specifying-environments

Since it's actually more code and we probably already have this global in a few places it's probably of low value. But I thought I'd pass it in on.

Reply:
Agreed - our current approach seems succinct and descriptive enough for now.